test/model/modelInfo/SimpleElement.test.ts

```diff
@@ -0,0 +1,10 @@
+import SimpleElement from "../../../src/model/modelInfo/SimpleElement";
+
+describe("SimpleElement", () => {
+  test("constructor", () => {
+    const el = new SimpleElement("name", "namespace", "typeName");
+    expect(el.name).toBe("name");
+    expect(el.namespace).toBe("namespace");
+    expect(el.typeName).toBe("typeName");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)